app/src/main/java/com/cp/tasky/ui/sharecomponent/ClickableText.kt

```diff
@@ -0,0 +1,77 @@
+package com.cp.tasky.ui.sharecomponent
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.toUpperCase
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
+fun PostClickableText(
+    modifier: Modifier = Modifier,
+    text: String,
+    clickableText: String,
+    onTextClick: () -> Unit
+) {
+    val nonClickableString by remember {
+        mutableStateOf(
+            text.substring(
+                0,
+                text.indexOf(clickableText)
+            )
+        )
+    }
+    val clickableString by remember { mutableStateOf(clickableText) }
```

Comment:
Same here, the clickable text is just a hardcoded string or one from string resources. There's no need to make it a state if it can't change.